semantic/src/site/globals/site.overrides

```diff
@@ -169,15 +170,58 @@ p strong {
   margin-bottom: 5px;
 }
 
+// Steps line (vertical box - layout)
+#application-area .line-vertical-box {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  min-height: 30px; // TODO: Need to be updated for mobile
+  align-items: center;
+  justify-content: space-between;
+}
+
+// Inner steps line (left side - icon & header)
+#application-area .line-vertical-box .left {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+// Steps line - header
 #application-area .ui.list > .item .header {
   color: @darkGrey;
   font-weight: @regularWeight;
   font-size: @fontMedium;
+  width: 200px;
+  overflow: hidden; // So long titles don't overflow the header area
 }
+
+// Steps line - icon
 #application-area .ui.list > .item > i.icon {
   color: @darkGrey;
 }
```

Comment:
Since we'll be making a lot of things "darkGrey", it would make sense to just make a generic class `.dark-grey`. Then we just have to apply `className=".dark-grey (etc...)" in place.

We should probably have a section in this file for all these "generic over-rides".

Reply:
I moved this to the `list.overwrite` since I think it something that should be used as default any icons showing on a list